controllers/ovncontroller_controller.go

```diff
@@ -659,7 +690,7 @@ func (r *OVNControllerReconciler) reconcileNormal(ctx context.Context, instance
 		}
 		instance.Status.Conditions.MarkTrue(condition.ServiceConfigReadyCondition, condition.ServiceConfigReadyMessage)
 	} else {
-		Log.Info("OVNController DaemonSet not ready yet. Configuration job cannot be started.")
+		Log.Info("OVNController and OVS DaemonSets not ready yet. Configuration job cannot be started.")
```

Comment:
nit: technically speaking You should say here "OVNController OR OVS DaemonSet not ready yet"